Sections/US-States/FL/GPP Extension: IAB Privacy’s Florida Privacy Technical Specification.md

```diff
@@ -0,0 +1,190 @@
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">SaleOptOutNotice</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Notice of the Opportunity to Opt Out of the Sale of the Consumer’s Personal Data&nbsp;</span><p></p><code>0</code> = Not Applicable<span style="color:rgb(36, 41, 47);">,
+          </span>The
+          <span style="color:rgb(36, 41, 47);">Controller does not Sell Personal Data</span>
+          <span style="color:rgb(36, 41, 47);"><p></p><code>1</code> = Yes, notice was provided</span>
+          <span style="color:rgb(36, 41, 47);"><p></p><code>2</code> = No, notice was not provided</span>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">TargetedAdvertisingOptOutNotice</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Notice of the Opportunity to Opt Out of Processing of the Consumer’s Personal Data for Targeted Advertising</span><p></p><code>0</code> = Not Applicable<span style="color:rgb(36, 41, 47);">, the Controller does not Process Personal Data for Targeted Advertising</span>
+          <span style="color:rgb(36, 41, 47);"><p></p><code>1</code> = Yes, notice was provided</span>
+          <span style="color:rgb(36, 41, 47);"><p></p><code>2</code> = No, notice was not provided</span>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">SaleOptOut</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Opt-Out of the Sale of the Consumer’s Personal Data</span><p></p><code>0</code> = Not Applicable, SaleOptOutNotice value was not applicable or no notice was provided<p></p><code>1</code> = Opted Out<p></p><code>2</code> = Did Not Opt Out</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">TargetedAdvertisingOptOut</span>
+        </td>
+        <td>Int(2)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Opt-Out of Processing the Consumer’s Personal Data for Targeted Advertising</span><p></p><code>0</code> = Not Applicable, TargetedAdvertisingOptOutNotice value was not applicable or no notice was provided<p></p><code>1</code> = Opted Out<p></p><code>2</code> = Did Not Opt Out</td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">SensitiveDataProcessing</span>
+        </td>
+        <td>Bitfield(16)</td>
+        <td>Two bits for each Data Activity:<p></p><code>0</code> = Not Applicable, the Controller does not Process the specific category of Sensitive Data<p></p><code>1</code> = Consent <p></p><code>2</code> = No Consent&nbsp;<span style="color:rgb(36, 41, 47);"><p></p>(1). Consent to Process the Consumer’s Sensitive Data Consisting of Personal Data Revealing Racial or Ethnic Origin.</span><p></p>
```

Comment:
In previous State sensitive data handling VA for example "2" equals consent, why change it to 1 in MT, OR, & TX?

Reply:
Confirming we'll revise once public comment closes to keep consistent across state sections.